src/Builder/FrontendDialectTransformer.hpp

```diff
@@ -55,6 +55,16 @@ struct ImportOptions {
   //   - (arg0: tensor<3x4x5xf32>, arg1: tensor<10x5xf32>)
   //
   std::string shapeInformation = "";
+  // Custom onnx.dim_params attributes for the graph inputs for specifying
+  // relationship among their ranks.
+  // Its format is 'input_id:dim=sym,dim=sym,dim=sym|input_id:dim=sym,dim=sym..'
```

Comment:
`dim_id` would be better than `dim`

Reply:
`..` at the end would be three dots `...`? Also that `...` is for `|` meaning multiple `input_id:dim=sym,dim=sym,dim=sym`, or for `,` meaning multiple `dim=sym,dim=sym`?

Reply:
Thanks. Fixed.